src/customTheme.ts

```diff
@@ -101,36 +103,81 @@ const tertiaryTextDarkmode = coolGray['400']
 // ***** Collect constants, put them in a customTheme *****
 
 const themeConfig: ThemeConfig = {
-  initialColorMode: 'light',
   useSystemColorMode: !!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE,
 }
 
-const customTheme = extendTheme({
-  colors: {
-    blue: blueColor,
-    primary: primaryColor,
-    secondary: blueColor,
-    coolGray,
-    fuchsia,
-    sky,
-    primaryTextLightmode,
-    secondaryTextLightmode,
-    tertiaryTextLightmode,
-    primaryTextDarkmode,
-    secondaryTextDarkmode,
-    tertiaryTextDarkmode,
-  },
-  fonts: {
-    heading: '"Inter", sans-serif;',
-    body: '"Inter", sans-serif;',
-    mono: '"Roboto Mono", monospace;',
-  },
-  styles: {
-    global: {
-      'html, body, #__next': { height: '100%' },
+const customTheme = extendTheme(
+  {
+    colors: {
+      blue: blueColor,
+      primary: primaryColor,
+      secondary: blueColor,
+      coolGray,
+      fuchsia,
+      sky,
+      primaryTextLightmode,
+      secondaryTextLightmode,
+      tertiaryTextLightmode,
+      primaryTextDarkmode,
+      secondaryTextDarkmode,
+      tertiaryTextDarkmode,
+    },
+    fonts: {
+      heading: '"Inter", sans-serif;',
+      body: '"Inter", sans-serif;',
+      mono: '"Roboto Mono", monospace;',
+    },
+    styles: {
+      global: {
+        'html, body, #__next': { height: '100%' },
+      },
+    },
+    config: themeConfig,
+    components: {
+      Link: {
+        baseStyle: (props: Dict) => {
+          return { color: mode('primary.500', 'primary.200')(props) }
+        },
+      },
+      Button: {
+        variants: {
+          cta: (props: Dict) => {
+            return {
+              fontWeight: 'black',
+              fontSize: '2xl',
+              letterSpacing: 'tight',
+              p: 6,
+              size: 'lg',
+              boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)',
+              borderRadius: '2xl',
+              bg: mode('primary.200', 'primary.900')(props),
+              color: mode('primary.900', 'white')(props),
+              _hover: {
+                bg: mode('primary.100', 'primary.700')(props),
+                cursor: 'pointer',
+              },
+              _active: {
+                bg: mode('primary.200', 'primary.800')(props),
+                boxShadow: '0px 2px 2px rgba(0, 0, 0, 0.25) !important',
+              },
+            }
+          },
+        },
+      },
+      Container: {
+        variants: {
+          fluid: {
+            maxWidth: 'container.xl',
+          },
+        },
+      },
     },
   },
-  config: themeConfig,
-})
+  withDefaultColorScheme({ colorScheme: 'primary' }),
+  withDefaultColorScheme({
+    colorScheme: 'gray',
+    components: ['Code', 'BlockQuote'],
+  })
```

Comment:
This is really convenient: I've set the default color scheme to primary except for Code and BlockQuote components. Everything seems fine, but let me know if you see any component with the wrong default color scheme now.

Reply:
The code and blockquotes look the same to me, so they are okay to go. Link color needs fixing though. 😉